components/VM_Arm/configurations/vm.h

```diff
@@ -68,22 +68,28 @@
     attribute int num_extra_frame_caps; \
     attribute int extra_frame_map_address; \
     attribute { \
-        string linux_ram_base; \
-        string linux_ram_paddr_base; \
-        string linux_ram_size; \
-        string linux_ram_offset; \
+        string ram_base; \
```

Comment:
Since we are doing a breaking change here anyway, I wonder if we should also change `string`to `uint64_t` here. There seems no point why these are string, except that in the past this was necessary because `uint64_t` did not exist.